unreleased/metricstransform-remove-deprecated-config-option.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: "metricstransformprocessor"
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove deprecatedetric_me settings option
+
+# One or more tracking issues related to the change
+issues: [12601]
```

Comment:
This is not a correct issue

Reply:
My bad, https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12740